src/components/Toast/types.tsx

```diff
@@ -0,0 +1,24 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactChild } from 'react';
+
+export type ToastProps = {
+    isOpen: boolean;
+    style: ToastStyles;
+    icon: ReactChild;
+    children?: ReactChild;
+};
+
+export enum ToastStyles {
+    Information = 'Information',
+    Warning = 'Warning',
+    Tip = 'Tip',
+    Note = 'Note',
```

Comment:
```suggestion
    Loud = 'Loud',
    Danger = 'Danger',
    Success = 'Success',
    Warning = 'Warning',
```

Reply:
The idea was to have the same color option as the Tooltip: Purple, green, red and yellow. But I agree with that naming. Design in Figma as a Fondue component is not done yet and I believe there might be a possibility the colors won't be that bright.